### ISSUE TITLE
How to modify the dataset path

### ISSUE THREAD

**Issue by Gaozzzz** [peteanderson80/Matterport3DSimulator#136]
self.sims[i].newEpisode([scanId], [viewpointId], [heading], [0])
ValueError: MatterSim: Could not open skybox RGB files at: ./data/v1/scans//sT4fr6TAbpF/matterport_skybox_images/6e41a7632c5a4048a17a316d7192b97e_skybox_small.jpg

I think the cause of this error is a bug with './data/v1/scans/', how can I modify this part

**Comment by jdiazram** [comment]
@Gaozzzz I have the same problem. Could you solve it?

**Comment by Gaozzzz** [comment]
@jdiazram sorry, I have not solve it yet

**Comment by jdiazram** [comment]
@Gaozzzz I solved this by reinitiating all (cloning, docker, everything XD), using the docker, and creating another directory with repo matterport3Dsimulator, installing the minimum for my problem with pip3. And now it's working fine so far.

**Comment by Gaozzzz** [comment]
@jdiazram Thank you very much for your detailed guide. Good luck with your research.

**Comment by SuneoLi** [comment]
Yes, the cause of this error is **the bug with the default data root** './data/v1/scans/'.

To fix it, I used the following steps...

1.  Open the Matterport3DSimulator project with **VScode**, then **Ctrl + Shift + F** to open the search bar and enter the original default path "**data/v1/scans**". (PS: Do not enter the “/” before “data”);
 ![e85663664e8b3b528923c5c7a6bdc1c](https://github.com/user-attachments/assets/a74cc54f-62ab-463f-9271-ae81a788501a)
2.  Replace your search with your own dataset path, for instance, **change all "data/v1/scans"** to "/xxx/xxx/v1/scans/";
3.  **Delete** the folder "build";
 `rm -rf ./build/`
4.  Re-execute **cmake** (These two lines are the same command);
`cmake -D OPENGL_opengl_LIBRARY=/usr/lib/x86_64-linux-gnu/libGL.so -D PYTHON_EXECUTABLE=/home/xxx/.conda/envs/xxx/bin/python3 -DEGL_RENDERING=ON ..`
5.  Re-execute **make**;
`make -j8`
6.  Copy the two compiled _**.so**_ files into the conda environment's library; 
`cp MatterSim.cpython-36m-x86_64-linux-gnu.so /home/xxx/.conda/envs/xxx/lib/python3.6/site-packages`
`cp libMatterSim.so /home/xxx/.conda/envs/xxx/lib/python3.6/site-packages`
7.  Rerun the test.
`./build/tests ~Timing`
![4a44ae8598613742e4c5112c79243d3](https://github.com/user-attachments/assets/355d7e7a-566f-47e3-aab3-d1fb25699d6a)

Please note that your paths and mine may differ.
